### PR TITLE
Tidy syntax node

### PR DESCRIPTION
**Pull request by AdamSpeight2008**
[Compiler VB SyntaxNode] Tidy Up
- Remove redundant `Me.`,`MyBase`, Casts, Namespace etc

### CODE REVIEW

**Comment by davkean** [comment]
@jaredpar Why didn't code formatter catch these?

**Comment by jaredpar** [comment]
@dakean I haven't run the codeformatter in a while because of schedule pressures.  I have it on the agenda to run again this week.

File: src/Compilers/Core/MSBuildTask/Portable/project.lock.json [hunk 1]
```diff
@@ -468,6 +468,24 @@
           "lib/net45/_._": {}
         }
       },
+      "System.Collections.Immutable/1.1.38-beta-23428": {
```
Comment (gafter) [inline src/Compilers/Core/MSBuildTask/Portable/project.lock.json]: You should not be touching these files unless you've changed the project refererences.

File: src/Compilers/VisualBasic/Portable/Syntax/VisualBasicSyntaxNode.vb [hunk 2]
```diff
@@ -205,28 +205,28 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             If s_serializationData Is Nothing Then
                 Dim data = New Object() {
                     GetType(Object).GetTypeInfo().Assembly.FullName,
-                    GetType(Microsoft.CodeAnalysis.DiagnosticInfo).GetTypeInfo().Assembly.FullName,
```
Comment (gafter) [inline src/Compilers/VisualBasic/Portable/Syntax/VisualBasicSyntaxNode.vb]: This was much more clear with the full type names included.

File: src/ExpressionEvaluator/Core/Test/ResultProvider/project.lock.json [hunk 3]
```diff
@@ -3,6 +3,23 @@
   "version": 1,
   "targets": {
     ".NETFramework,Version=v4.5": {
+      "Microsoft.CodeAnalysis.Test.Resources.Proprietary/1.2.0-beta-20151016-11": {
```
Comment (gafter) [inline src/ExpressionEvaluator/Core/Test/ResultProvider/project.lock.json]: You should not be touching these files.

**Comment by gafter** [comment]
@dotnet/roslyn-compiler FYI.

I'd rather we hold off on this until after @jaredpar  has run the code formatter.

**Comment by TyOverby** [comment]
Agree with @gafter

**Pull request closed without merging**